src/gfn/modules.py

```diff
@@ -240,13 +240,20 @@ def __init__(
         self.conditioning_module = conditioning_module
         self.final_module = final_module
 
-    def forward(
-        self, states: States, conditioning: torch.tensor
+    def _forward_trunk(
```

Comment:
is what you call `trunk` the same thing I called `torso` before ?

Reply:
Yeah -- let me unify the naming